src/test/java/org/xstefank/TestUtils.java

```diff
@@ -29,10 +34,27 @@ public static FormatConfig loadFormatFromYamlFile(String fileName) {
         }
     }
 
+    public static boolean fileHasElement(File file, String element) {
+        try (FileReader fileReader = new FileReader(file);
+             BufferedReader bufferedReader = new BufferedReader(fileReader)) {
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                if (line.contains(element)) {
+                    return true;
+                }
+            }
+        } catch (IOException e) {
+            throw new IllegalStateException("Cannot read file " + file.getName(), e);
+        }
+        return false;
+    }
+
+
     private static JsonNode loadJson(String fileName) {
         try {
             File file = new File(TestUtils.class.getClassLoader().getResource(fileName).getFile());
-            return new ObjectMapper().readTree(file);
+            JsonNode jNode = new ObjectMapper().readTree(file);
+            return Utils.reformatPayload(jNode);
```

Comment:
I am not sure about this. If something starts failing in `reformatPayload` a lot of not associated tests can start failing